market/connector/microsoft365/calendar/product.json

```diff
@@ -0,0 +1,51 @@
+{
+    "properties": {
+        "tags": "me.user,me.calendar,users.calendar,me.Actions",
+        "scopes": "User.Read Calendars.ReadWrite",
+        "demo": "Calendar",
+        "main": "../msgraph/"
+    },
+    "installers": [
+        {
+            "...": "<../msgraph/project-res.json>"
+        },
+        {
+            "...": "<../msgraph/rest-client.json>"
+        },
+        {
+            "id": "project-resource",
+            "data": {
+                "dialog": {
+                    "source": "@main@demo/msGraphDemo/src_hd",
```

Comment:
ahh I see ... I already inventend that other pattern :-(   :dancers:

Reply:
then it loooks to me as if the designer is already aware of that featureset ... and we can keep it as is. sorry for the confusion

Reply:
ahh but the pattern is with closing tag I think